packages/api/cms-api/src/mailer/mailer.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Inject, Injectable, Logger } from "@nestjs/common";
+import { createTransport, Transporter } from "nodemailer";
+import { Address, Options as MailOptions } from "nodemailer/lib/mailer";
+
+import { MAILER_MODULE_OPTIONS } from "./mailer.constants";
+import { MailerModuleConfig } from "./mailer.module";
+
+@Injectable()
+export class MailerService {
+    private mailerTransport: Transporter;
+    private readonly logger = new Logger(MailerService.name);
+
+    constructor(@Inject(MAILER_MODULE_OPTIONS) private readonly mailerConfig: MailerModuleConfig) {
+        this.mailerTransport = createTransport(mailerConfig);
```

Comment:
you should not create services in an service, instead move createTransport into the module and inject the transport object

Reply:
Since #3969 was closed, I should probably include this part of the change in the current pull request.

Reply:
mailerTransport isn't really a service though. IMO this is fine. It's also similar to this: https://github.com/vivid-planet/comet/blob/next/packages/api/cms-api/src/blob-storage/backends/blob-storage-backend.service.ts#L17-L25

Reply:
Dependencies should be injected. I don't see any downside with doing that.

(BlobStorageBackendService should better also get the backend injected instead of creating it)

Reply:
I'm also fine with injecting it.

Reply:
done :-)